Test approved label

realstealthninja opened the pull request:
#### Description of Change
<!--
Thank you for your Pull Request. Please provide a description above and review
the requirements below.

Contributors guide: https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md
-->

#### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes: <!-- Please add a one-line description for developers or pull request viewers -->

realstealthninja committed:
ci: use realstealthninja/label-when-approved

realstealthninja committed:
Merge branch 'TheAlgorithms:master' into approved-label

realstealthninja committed:
ci: use latest version

realstealthninja committed:
ci: use v0.4.0

realstealthninja committed:
ci: rename secret to token

realstealthninja committed:
ci: use with instead of env

realstealthninja committed:
ci: use secret instead of token

realstealthninja committed:
chore: bump workflow to 0.4.2

realstealthninja committed:
ci: update approved-label to latest

realstealthninja committed:
ci: use main ref

stealthninja-alt reviewed (changes requested):
remove approval test

vil02 committed:
style: resolve `-Wunused-parameter` (#2990)

vil02 committed:
style: remove unused variable (#2991)

9santa committed:
doc: fix a typo in ccl.h(#2993)

Fixed "Cicular" -> "Circular"

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

vil02 committed:
fix: remove potential segmentation fault from `factorial_memoization.cpp` (#2994)

stealthninja-alt reviewed (changes requested):
test 4

vil02 committed:
fix: remove potential segv from `postfix_evaluation.cpp` (#2995)

git5v committed:
Added a bit manipulation program to determine even or odd numbers (#2959)

* Create check_even_odd.cpp

Implementation to Check if a number is Even or Odd using Bitwise Operator

* Update check_even_odd.cpp

* Create factorial_top_down_dp.cpp

* Delete dynamic_programming/factorial_top_down_dp.cpp

Deleted the one file as there was 2 files in the commit

* Create factorial_top_down_dp.cpp

* Delete dynamic_programming/factorial_top_down_dp.cpp

* Update bit_manipulation/check_even_odd.cpp

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

* Update bit_manipulation/check_even_odd.cpp

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

* Update bit_manipulation/check_even_odd.cpp

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

* Update bit_manipulation/check_even_odd.cpp

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

* Update check_even_odd.cpp

modified

* Update check_even_odd.cpp

reverted to boolean to check if number is even or not

---------

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

vil02 committed:
refactor: input size should not be a template argument in `postfix_evaluation` (#2996)

vil02 committed:
fix: handle wrong inputs in `postfix_evaluation` (#3005)

Rudra2637 committed:
Fix: use uint64_t for input and counter in countSetBits (#3003)

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

vil02 committed:
style: remove interaction with the user (#3006)

Helps with #2753

Co-authored-by: realstealthninja <68815218+realstealthninja@users.noreply.github.com>

realstealthninja committed:
chore: fix file formatter (#3007)

* chore: fix file formatter

* chore: remove else and make it a guard clause as it was initally

realstealthninja committed:
ci: bump codeql to v3 (#3008)

Rudra2637 committed:
correct grammar and wording in contribution guidelines (#3012)

realstealthninja committed:
Revert "style: remove interaction with the user (#3006)"

This reverts commit bde09c2db9821f3afc419f1f4ff476eea1ad8efa.

realstealthninja committed:
Revert "chore: fix file formatter (#3007)"

This reverts commit 36e6405a301a67f121e0b8baff1f0afa8d28b531.

realstealthninja committed:
Revert "correct grammar and wording in contribution guidelines (#3012)"

This reverts commit a4d41933e927414664e9f1156de6bca37767c6b9.

realstealthninja committed:
Revert "Added a bit manipulation program to determine even or odd numbers (#2959)"

This reverts commit bcb09dff24bf42586b90cc3fc0c9664df708e712.

realstealthninja committed:
Revert "fix: remove potential segv from `postfix_evaluation.cpp` (#2995)"

This reverts commit 379a7fceb88383978419df882be1e740f8dce964.

realstealthninja committed:
Revert "fix: remove potential segmentation fault from `factorial_memoization.cpp` (#2994)"

This reverts commit 1426fa42fb95dd39b0270e205e0c168b20cbcaf4.

realstealthninja committed:
Revert "style: resolve `-Wunused-parameter` (#2990)"

This reverts commit 335034743918b94e05ae89f232d27fe181dfd56f.

realstealthninja committed:
Revert "Fix: use uint64_t for input and counter in countSetBits (#3003)"

This reverts commit a2efdcb787946c5b3bcc8b37f33f2bb10701e7ce.

realstealthninja committed:
Revert "ci: bump codeql to v3 (#3008)"

This reverts commit 320d68c31c1c6708cfc1bcd0610bac424ed92125.

realstealthninja committed:
Revert "doc: fix a typo in ccl.h(#2993)"

This reverts commit af48da59e8cb9a26f6156fe2e8bd4423e0e5a46f.

realstealthninja committed:
Revert "refactor: input size should not be a template argument in `postfix_evaluation` (#2996)"

This reverts commit b6d0448021a9196ecbe074a938cef9d5716b4d43.

realstealthninja committed:
Revert "fix: handle wrong inputs in `postfix_evaluation` (#3005)"

This reverts commit 2ea98ce059ce7cebc416cd944abf6939f4d4d14c.

realstealthninja committed:
Revert "style: remove unused variable (#2991)"

This reverts commit 079d143fbe3e8a3594b39235c76bc3305c6bf8c4.
